_posts/2016-08-27-mouse-dna-magnetic-fields.md

```diff
@@ -0,0 +1,48 @@
+---
+layout: post
+title: Low magnetic fields leave mouse DNA undamaged
+author: RaoOfPhysics
+image:
+  feature: 2016-08-27-mouse-dna-magnetic-fields.jpg
+  credit: Nogas1974
+  caption: “DNA com GGN”
+---
+
+_Cells in brain, kidney and liver, which are involved in the transport, reabsorption or storage of iron, withstand sustained exposure to magnetic fields._
+
+The 19th-century works of Hans Christian Ørsted, Michael Faraday, James Clerk Maxwell and others demonstrated that electricity and magnetism are in fact two sides of the same coin: a magnet moving near a coil of wires induces a current in the coil and circulating electric current generates magnetic fields.
+We have since harnessed this unified electromagnetism to provide for the energy needs of billions around the globe.
+However, it is thought that prolonged exposure to ambient electromagnetic (EM) fields can affect certain cells in the body, causing a variety of cancers.
+For this reason, several countries have adopted limits for EM exposure, with most European countries limiting fields to 0.1&nbsp;mT ([milliteslas](https://en.wikipedia.org/wiki/Tesla_(unit))).
+
+Researchers tested the effects of EM fields on biological cells in a controlled environment by exposing two groups of mice to low-frequency fields of 0.1&nbsp;mT and 1.0&nbsp;mT respectively over eight weeks, while a third group of mice living in similar conditions but without the EM exposure served as control.
+The frequency chosen for the research was 50&nbsp;Hz, which is the frequency in most countries of the transmission of electricity via power lines.
+The same researchers previously found that an eight-week exposure to a higher field of 1.5&nbsp;mT caused unrepaired DNA damage in mice.
+In both these studies, the scientists used breaks in the DNA strands in the cell nuclei as a measure of DNA damage.
+In the latest study, they also looked for signs that DNA inside mitochondria was being produced at lower rates than usual.
+
+<!--break-->
+
+Previous research has suggested that extended exposure to EM fields damages cells that are in some way related to the transport, reabsorption or storage of iron.
+The researchers therefore studied how cells involved in these activities located in the brain, the kidney and the liver were affected by EM exposure.
+
+![“DNA com GGN”]({{ site.url }}/images/2016-08-27-mouse-dna-magnetic-fields.jpg)
+
+> _“DNA com GGN”. [Image](https://commons.wikimedia.org/wiki/File:DNA_com_GGN.jpg) by Nogas1974 / [CC BY-SA 4.0](https://creativecommons.org/licenses/by-sa/4.0/)_
+
+Three sets of mice were kept in identical conditions differing only in the EM fields inside their cages.
```

Comment:
kept in otherwise identical environments but were exposed to different EM fields over eight weeks: one group at the low field level of 0.1 mT, one group at 1.0 mT and the third group with no radiation.